.github/workflows/build.yml

```diff
@@ -227,11 +227,11 @@ jobs:
           chmod +x ace2/ace
           chmod +x openfpga/openfpga
```

Comment:
I think in the distribution share folder where yosys-plugin so and other files are present, should also be part of distribution.